runtime/src/lib.rs

```diff
@@ -109,6 +109,19 @@ impl module_evm::Config for Config {
     const CONFIDENTIAL: bool = true;
 }
 
+#[allow(clippy::declare_interior_mutable_const)]
+impl modules::access::Config for Config {
+    const METHOD_AUTHORIZATIONS: Lazy<modules::access::types::Authorization> = Lazy::new(|| {
+        modules::access::types::Authorization::with_filtered_methods([(
+            "evm.Create",
+            modules::access::types::MethodAuthorization::allow_from([Address::from_bech32(
+                "oasis1qryqqccycvckcxp453tflalujvlf78xymcdqw4vz",
```

Comment:
It would be nice to have a comment containing details about this address (e.g. what is this address, who is the owner). One could also move this address to a constant if it will be used multiple times.

Reply:
This specific address is from one of the unit tests, but yes, I'll add a comment once we have the actual address.